index.html

```diff
@@ -6,7 +6,33 @@
     <title>To Do</title>
     <link rel="stylesheet" href="css/style.css" />
   </head>
+
   <body>
+    <header>
+      <div class="navbar">
+        <h1>TO DO</h1>
+      </div>
+    </header>
+
+    <main>
+      <section class="container" id="to-do-list">
```

Comment:
We use camelCase for id in javascript. Also I would suggest following the Figma design format. But, it is not necessary to add proper CSS for these PRs.
```suggestion
      <section class="container" id="toDoList">
```

Reply:
@sku-wtag Done, bhai.